src/compiler/checker.ts

```diff
@@ -45308,7 +45308,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             }
 
             meaning |= SymbolFlags.Alias;
-            const entityNameSymbol = isEntityNameExpression(name) ? resolveEntityName(name, meaning) : undefined;
+            const entityNameSymbol = isEntityNameExpression(name) ? resolveEntityName(name, meaning, /*ignoreErrors*/ true) : undefined;

```

Comment:
It's my understanding that we expect that this `resolveEntityName` call might fail, judging from the if statement below and the rest of the code (and an earlier resolve call that also ignores errors)`, so we shouldn't report errors here.